packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -559,8 +533,8 @@ class SystemRulesTable extends React.Component {
                         onCollapse={ this.onCollapse }
                         onSort={ this.onSort }
                         sortBy={ sortBy }
-                        onSelect={ (currentRows.length !== 0) ? this.onSelect : () => null }
-                        rows={ (currentRows.length === 0) ? emptyRows : currentRows }>
+                        onSelect={ (remediationsEnabled && currentRows.length !== 0) ? this.onSelect : undefined }
```

Comment:
Passing undefined could cause issues. The `Table` component could still try to call what is passed to `onSelect` and result in an unwanted error or warning. Using the mute function is not optimal. Optimal would be to only pass the prop if it's set, that would require to prepare the props before hand and spread them in.